pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java

```diff
@@ -257,6 +257,47 @@ public String loadParticipantData(Handle handle, JsonElement datstatData, JsonEl
         return pepperUser.getUserGuid();
     }
 
+    public ActivityInstanceDto createPrequal(Handle handle, String participantGuid, long studyId, String ddpCreated,
+                                             JdbiActivity jdbiActivity,
+                                             ActivityInstanceDao activityInstanceDao,
+                                             ActivityInstanceStatusDao activityInstanceStatusDao,
+                                             AnswerDao answerDao) throws Exception {
+
+        Long studyActivityId = jdbiActivity.findIdByStudyIdAndCode(studyId, "PREQUAL").get();
+        Instant instant;
+        try {
+            instant = Instant.parse(ddpCreated);
+        } catch (DateTimeParseException e) {
+            throw new Exception("Could not parse required createdAt value for prequal, value is " + ddpCreated);
+        }
+        long ddpCreatedAt = instant.toEpochMilli();
+        ActivityInstanceDto dto = activityInstanceDao
+                .insertInstance(studyActivityId, participantGuid, participantGuid, InstanceStatusType.CREATED,
+                        true,
+                        ddpCreatedAt,
+                        null, null, null);
+
+        //populate PREQUAL answers
+        JdbiProfile jdbiProfile = handle.attach(JdbiProfile.class);
+        UserProfileDto profileDto = jdbiProfile.getUserProfileByUserGuid(participantGuid);
+        answerTextQuestion("PREQUAL_FIRST_NAME", participantGuid, dto.getGuid(),
+                profileDto.getFirstName(), answerDao);
+
+        answerTextQuestion("PREQUAL_LAST_NAME", participantGuid, dto.getGuid(),
+                profileDto.getLastName(), answerDao);
+
+        List<SelectedPicklistOption> options = new ArrayList<SelectedPicklistOption>();
+        options.add(new SelectedPicklistOption("DIAGNOSED"));
+        answerPickListQuestion("PREQUAL_SELF_DESCRIBE", participantGuid, dto.getGuid(),
+                options, answerDao);
+
+        //add 30seconds to created_date and populate complete status
```

Comment:
Is 30 seconds safe? Should it be some time after user is created but before the About You is created?

Reply:
We took a closer look at this -- it doesn't seem to have any know side effects so this way should be fine.

Reply:
Basically I am using participant created date so that the prequal activity creation date is close to reality rather than current system date. 
To avoid DB constraint violation while inserting second status row for complete status, I had to change the timestamp, hence added 30seconds. 
These dates are not used any where, we should be fine.